product_docs/docs/pem/8/changing_default_port.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: "Changing the default port"
+---
+
+The PEM Server is configured after the installation either automatically (in Windows) or manually by running the configuration script (in Linux). While configuring the PEM Server, by default the 8443 port is assigned for the web services. You can change the port as per your requirement after configuration by changing a few parameters in the apache configuration files. The apache configuration files names and locations are platform specific.
+
+## For RHEL and CentOS
+
+```text
```

Comment:
```suggestion
```ini
```
Verify that the formatting works after making this update, but typically you want to use ini for config files or any files that are made up of key:value pairs

Reply:
That said, this code block includes updates to a config file as well as other optional commands. Unless there is a reason to do this as one code block, I would break this up and pull the comments out as regular text.

Reply:
I agree with Dee Dee. What you have as comments should be regular text.

Reply:
I have break up this code block and have used ini, text and shell